src/Shell/Task/TwbsAssetsTask.php

```diff
@@ -38,7 +38,7 @@ public function __construct()
     public function installAssets()
     {
         $this->info('Checking npm...');
-        if (!`which npm`) {
+        if (strtoupper(substr(PHP_OS, 0, 3)) !== 'WIN' && !`which npm`) {
```

Comment:
Maybe use `where.exe npm` on Windows instead?

Reply:
We could also execute ``npm --version`` and check the return value.